src/test/java/org/folio/search/controller/SearchInstanceIT.java

```diff
@@ -420,4 +429,14 @@ private static Stream<Arguments> testDataProvider() {
       arguments("(title all \"{value}\")", "A semantic web primer : 0747-0850")
     );
   }
+
+  private static Stream<Arguments> testIssnDataProvider() {
+    return Stream.of(
+      arguments("issn = {value}", "0040-781X"),
+      arguments("issn = {value}", "0040-781x"),
+      arguments("issn = {value}", "*0-781X"),
+      arguments("issn = {value}", "*0-781x"),
+      arguments("issn = {value}", "0040-781*")
```

Comment:
Could you check if next cases, that aren't described in requirements, will work:
- an * in a middle of the search term
- two or more * in the search term

Reply:
Added a lot of cases.

Reply:
Could you please add a comment to the issue with some of the examples, that are not described in requirements. It will be helpful for POs to know about it

Reply:
OK